Week4/BlockbusterRecycler/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,29 @@
 <resources>
     <string name="app_name">BlockbusterRecycler</string>
+    <string name="summaryTerminator">In 1984, a human soldier is tasked to stop an indestructible cyborg killing machine, both sent from 2029, from executing a young woman, whose unborn son is the key to humanity\'s future salvation.</string>
```

Comment:
Nice job, in the future this information will come from a database or a networking source, but for the purpose of this app it's fine to have it here.